Matey/Controllers/HomeViewController.swift

```diff
@@ -10,88 +10,97 @@ import UIKit
+        labelLend.text = "Lend"
+        labelLend.font = UIFont.systemFont(ofSize: 16)
+        labelLend.textColor = .secondaryLabel
+        labelLendAmount.text = "10,000 ₺"
+        labelLendAmount.font = UIFont.boldSystemFont(ofSize: 24)
+        labelLendAmount.textColor = UIColor(displayP3Red: 47/255, green: 140/255, blue: 79/255, alpha: 1.0)
+        labelBorrow.text = "Borrow"
+        labelBorrow.font = UIFont.systemFont(ofSize: 16)
+        labelBorrow.textColor = .secondaryLabel
+        labelBorrowAmount.text = "1,600 ₺"
+        labelBorrowAmount.font = UIFont.boldSystemFont(ofSize: 24)
+        labelBorrowAmount.textColor = .red
+        labelHeader.text = "Where is my Money?"
+        labelHeader.font = UIFont.boldSystemFont(ofSize: 24)
+        labelHeader.textColor = UIColor(displayP3Red: 47/255, green: 140/255, blue: 79/255, alpha: 1.0)
+    }
+
+    func setupPlusButton() {
+        plusButton = UIButton(frame: CGRect(x: 0, y: 0, width: 64, height: 64))
+        var menuButtonFrame = plusButton.frame
         menuButtonFrame.origin.y = view.bounds.height - (view.bounds.height / 5)
-          menuButtonFrame.origin.x = view.bounds.width/10 * 8
-          plusButton.frame = menuButtonFrame
-
-          plusButton.backgroundColor = UIColor(displayP3Red: 47/255, green: 140/255, blue: 79/255, alpha: 1.0)
-          plusButton.layer.cornerRadius = menuButtonFrame.height/2
-          view.addSubview(plusButton)
-
-          plusButton.setImage(UIImage(named: "plus"), for: .normal)
-          plusButton.imageView?.contentMode = .scaleToFill
-          plusButton.imageEdgeInsets = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
-          plusButton.addTarget(self, action: #selector(plusButtonAction(sender:)), for: .touchUpInside)
-
-          view.layoutIfNeeded()
-          plusButton.bringSubviewToFront(self.view)
-      }
-
-  // MARK: - Actions
-
-  @objc private func plusButtonAction(sender: UIButton) {
-      print("Tapped")
-      let storyboard = UIStoryboard(name: "History", bundle: nil)
-      guard let viewController = storyboard.instantiateViewController(withIdentifier: "HistoryViewController") as? HistoryViewController else { return }
-      self.navigationController?.pushViewController(viewController, animated: true)
-      }
+        menuButtonFrame.origin.x = view.bounds.width/10 * 8
+        plusButton.frame = menuButtonFrame
+
+        plusButton.backgroundColor = UIColor(displayP3Red: 47/255, green: 140/255, blue: 79/255, alpha: 1.0)
+        plusButton.layer.cornerRadius = menuButtonFrame.height/2
+        view.addSubview(plusButton)
+
+        plusButton.setImage(UIImage(named: "plus"), for: .normal)
+        plusButton.imageView?.contentMode = .scaleToFill
+        plusButton.imageEdgeInsets = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
+        plusButton.addTarget(self, action: #selector(plusButtonAction(sender:)), for: .touchUpInside)
```

Comment:
```suggestion
        plusButton.addTarget(self, action: #selector(plusButtonAction()), for: .touchUpInside)
```
Writing it this way will increase readability.